src/Calculator/Views/DateCalculator.xaml.cpp

```diff
@@ -102,7 +102,7 @@ void DateCalculator::FromDate_DateChanged(_In_ CalendarDatePicker^ sender, _In_
     {
         auto dateCalcViewModel = safe_cast<DateCalculatorViewModel^>(this->DataContext);
         dateCalcViewModel->FromDate = e->NewDate->Value;
-        TraceLogger::GetInstance().LogDateDifferenceModeUsed(ApplicationView::GetApplicationViewIdForWindow(CoreWindow::GetForCurrentThread()));
+        TraceLogger::GetInstance().LogDateDifferenceModeUsed();
```

Comment:
We should update the `LogDateDifferenceModeUsed` to use the `windowId` value.  For an example, see `TraceLogger::LogHypButtonUsed`.

Reply:
updated.